docs/pattan_sendgrid/PattanEmail.md

```diff
@@ -0,0 +1,26 @@
+Module pattan_sendgrid.PattanEmail
+==================================
+
+Classes
+-------
+
+`PattanEmail(config_json=None)`
+:   Useful SendGrid API abstraction for sending emails.
+    It includes a command line interface to construct the configuration file by querying the SendGrid API directly.
+
+    ### Methods
+
+    `send_personalized_template_email(self, personalization_list, template_id, sender='DEFAULT', asm_group='DEFAULT', ip_pool='DEFAULT')`
+    :   This function should be used when the email is unique for each recipient.
```

Comment:
Maybe "customized" rather than "unique"? Or does this mean when you're sending messages one at a time? Either way it's not entirely clear